packaging/helm-charts/helm3/Makefile

```diff
@@ -0,0 +1,48 @@
+PROJECT_NAME=helm-charts
+
+include ../../../Makefile.os
+
+RELEASE_VERSION ?= latest
+CHART_NAME=strimzi-access-operator
+CHART_PATH ?= ../helm3/strimzi-access-operator
+CHART_RENDERED_TEMPLATES_TMP=../../../target/charts
+CHART_RENDERED_TEMPLATES_INSTALL=../../install
+HELM_CLI=helm
+
+helm_clean:
+	rm -rfv $(CHART_RENDERED_TEMPLATES_TMP)
+	rm -rf strimzi-$(RELEASE_VERSION)/charts/
+	rm -f $(CHART_NAME)-*.tgz
+
+helm_lint:
+	# Linting Helm Chart
+	$(HELM_CLI) lint --debug $(CHART_PATH)
+
+helm_install:
+	# Copying rendered template files to: $(CHART_RENDERED_TEMPLATES_INSTALL)
+	mkdir -p $(CHART_RENDERED_TEMPLATES_TMP)
+	helm template --namespace strimzi-access-operator --output-dir $(CHART_RENDERED_TEMPLATES_TMP) $(CHART_PATH)
+	$(FIND) $(CHART_RENDERED_TEMPLATES_TMP) -type f -name '*.yaml' -exec $(SED) -i '/^---/d' {} \;
+	$(FIND) $(CHART_RENDERED_TEMPLATES_TMP) -type f -name '*.yaml' -exec $(SED) -i '/^# Source: /d' {} \;
+	$(FIND) $(CHART_RENDERED_TEMPLATES_TMP) -type f -name '*.yaml' \
+	    | xargs -IFILE $(CP) FILE $(CHART_RENDERED_TEMPLATES_INSTALL)
+	rm -rf $(CHART_RENDERED_TEMPLATES_TMP)
+
+helm_pkg: helm_clean helm_lint helm_install
```

Comment:
As far as I can tell this make goal is never executed by running top level make commands (e.g. when running `make all`, `make helm_install` or `make release_pkg`) so could we omit this here and only have this code in the top level Makefile? Or alternatively call this from the top level Makefile?